### DISCUSSION TITLE
form-Item 的 tip-type 值为toast或者message时要怎么引入组件呢

### DISCUSSION THREAD

**Discussion by MinCpopo** [TaleLin/lin-ui#1205] [求助] [answered]
看到文档里有写 
通过设置 form-Item 的 tip-type 值为 message 或 toast 或 text 修改提示方式。默认值为 toast。 注意： 使用 message 和 toast 的方式需要在配置文件中引用组件
但是不知道具体应该怎么引入

**Comment by juzi214032** [comment, marked as answer]
在页面 json 中引入：
```json
{
  "usingComponents": {
    "l-form": "/dist/form/index",
    "l-form-item": "/dist/form-item/index",
    "l-toast": "/dist/toast/index",
    "l-message": "/dist/message/index"
  }
}

```

**Comment by MinCpopo** [comment]
这个方式之前就试过了，还是提示“请在页面中引入toast组件”，这是我的wxml代码
<l-form name="store" l-form-btn-class="l-form-btn-class" bind:linsubmit="submit">
        <l-form-item label="门店信息" name="store_name" label-placement="column" tip-type="toast"
            rules="{{loginIdRules}}">
            <l-input id="store_name" value="{{store.name}}" hide-label show-row="{{false}}" />
        </l-form-item>
</l-form>

**Reply by MinCpopo** [reply]
代码粘不上去了，反正就是用的验证表单里的示例代码

**Reply by juzi214032** [reply]
页面 wxml 中需要写：
```html
<l-toast />
```

**Reply by MinCpopo** [reply]
<l-form-item label="社会渠道门店信息" name="store_name" label-placement="column" tip-type="toast"
            rules="{{loginIdRules}}">
            <l-input id="store_name" value="{{store.name}}" hide-label show-row="{{false}}" />
        </l-form-item>  
是要定义 tip-type="toast"

**Reply by juzi214032** [reply]
设置了 tip-type="toast" ，同时也要在页面中写：
```html
<l-toast />
```

**Reply by MinCpopo** [reply]
不好意思啊，半天没明白，但是我按照示例代码写的，用的标签是l-form-item，要是页面还要写<l-toast />不知道该加在哪里，我截了个图，不知道能不能看到
![image](https://user-images.githubusercontent.com/42403875/103113260-ecfc6480-4694-11eb-824e-5017195cc3e8.png)

**Reply by juzi214032** [reply]
放在第一行就好了

**Reply by MinCpopo** [reply]
试了下果然可以了，多谢！
